api.md

```diff
@@ -1,30 +1,33 @@
 # API
   ## Table of Contents
-- [HOC](#hoc)
-  - [`withApiData()`](#withapidata)
-- [Config](#config)
-  - [`configureApiData()`](#configureapidata)
-  - [`useRequestHandler()`](#userequesthandler)
-- [Actions](#actions)
-  - [`performApiRequest()`](#performapirequest)
-  - [`invalidateApiDataRequest()`](#invalidateapidatarequest)
-  - [`afterRehydrate()`](#afterrehydrate)
-- [Selectors](#selectors)
-  - [`getApiDataRequest()`](#getapidatarequest)
-  - [`getResultData()`](#getresultdata)
-  - [`getEntity()`](#getentity)
-- [Types and interfaces](#types-and-interfaces)
-  - [`NetworkStatus`](#networkstatus)
-  - [`ApiDataBinding`](#apidatabinding)
-  - [`ApiDataRequest`](#apidatarequest)
-  - [`EndpointParams`](#endpointparams)
-  - [`ApiDataEndpointConfig`](#apidataendpointconfig)
-  - [`ApiDataGlobalConfig`](#apidataglobalconfig)
-  - [`ApiDataConfigBeforeProps`](#apidataconfigbeforeprops)
-  - [`ApiDataConfigAfterProps`](#apidataconfigafterprops)
-  - [`ApiDataState`](#apidatastate)
-  - [`RequestHandler`](#requesthandler)
-  - [`HandledResponse`](#handledresponse)
+- [API](#api)
+  - [Table of Contents](#table-of-contents)
```

Comment:
Beetje vreemd om het document en de toc zelf in de toc te vermelden, of niet? Waarom is dat toegevoegd?